src/style/SearchBar.css

```
@@ -0,0 +1,42 @@
+.search-bar {
+  width: 100%;
+  display: flex;
+  padding: 15px 16px;
+  justify-content: space-between;
+  align-items: flex-start;
+  border-radius: 10px;
+  background: #f5f5f5;
```

관련이 있는 속성끼리 붙여 써주시는 게 좋습니다.
```suggestion
  display: flex;
  justify-content: space-between;
  align-items: flex-start;
  width: 100%;
  padding: 15px 16px;
  border-radius: 10px;
  background: #f5f5f5;
```